extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -24,6 +35,10 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 	function createProvider(logGroup, providerName, src, slotMap, extra) {
 		extra = extra || {};
 
+		if (extra.overrideSizesPerCountry && extra.overrideSizesPerCountry[country]) {
+			overrideSizes(slotMap, extra.overrideSizesPerCountry[country]);
```

Comment:
I'm not a big fan of helpers too (it's like a bag of code that belongs nowhere and we probably shouldn't overuse it). :+1: 
In this case, the log object doesn't hurt my eyes (especially we pass it all around anyway). 
I'm more concern about mixing 2 different configuration approaches inside one function: via parameters (logGroup, providerName, src, slotMap) and via object (extra). Maybe it would be worth to unify it?
Using object approach is less convenient  while writing the code, but more convenient while reading it. 
I'm thinking about something like:
```javascript
function createProvider(config) { ... }
```
and then using it like:
```javascript
provider = factory.createProvider({
  'logGroup': logGroup,
  'slotName': 'DirectGpt',
  ...
});
```
what do you think?